tests/models_test.py

```diff
@@ -91,3 +94,42 @@ def test_no_top_variable_shape_4(app, last_feature_dim):
     input_shape = (None, None, 4)
     model = app(weights=None, include_top=False, input_shape=input_shape)
     assert model.output_shape == (None, None, None, last_feature_dim)
+
+
+@parametrize
+def test_model_summary(app, last_feature_dim, capsys, snapshot):
+    input_tensor = keras.layers.Input(shape=(224, 224, 3))
+    model = app(weights=None, input_tensor=input_tensor)
+    lq.models.summary(model)
+    out, err = capsys.readouterr()
```

Comment:
`lq.models.summary()`  has a `print_fn` which I think should be used instead of capturing the stout which can be really flaky.